File: frontend_web/src/types/Appointment.ts
import { Offer } from "./Offer";
import { User } from "./User";

export interface Appointment{
    _id: string,
    status: string,
    date: string,
    offer: Offer,
    createdAt: string |null,
    updatedAt: string | null,
    user: User,
    guests: number,
    new: boolean
}

File: frontend_android/app/src/main/java/com/example/trenucizapamcenje/models/Appointment.kt
package com.example.trenucizapamcenje.models


data class Appointment(

    val _id: String,
    val status: String,
    val date: String,
    val offer: Offer,
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val user: User,
    val guests: Int
)
